NDTensors/Project.toml

```
@@ -1,7 +1,7 @@
 name = "NDTensors"
 uuid = "23ae76d9-e61a-49c4-8f12-3f1a16adf9cf"
 authors = ["Matthew Fishman <mfishman@flatironinstitute.org>"]
-version = "0.2.30"
+version = "0.2.31"
```

I meant this:
```suggestion
version = "0.3.0"
```
in order to indicate a breaking change according to [semantic versioning](https://pkgdocs.julialang.org/v1/compatibility/#compat-pre-1.0).

---

Thanks for the reference! I updated this, thanks!

---

Thank you for the reference, I updated this thanks!